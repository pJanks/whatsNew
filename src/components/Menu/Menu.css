button {
  font-size: 120%;
  height: 30px;
  margin: 20px;
  width: 130px;
}

button:hover {
  border: 1px solid blue;
  cursor: pointer;

}

aside {
  border-right: 2px solid #000;
  width: 15%;
}
