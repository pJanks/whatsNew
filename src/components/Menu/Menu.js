import React from 'react';
import './Menu.css'

const Menu = () => {

  return (<aside>
    <button>local</button>
    <button>entertainment</button>
    <button>health</button>
    <button>science</button>
    <button>technology</button>
    </aside>)
}




export default Menu;
