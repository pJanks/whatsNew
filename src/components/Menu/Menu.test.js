import React from 'react';
import ReactDOM from 'react-dom';
import Menu from './Menu';
import { shallow } from 'enzyme';

describe('Menu', () => {
  let wrapper, instance;

  beforeEach(() => {
    wrapper = shallow(<Menu changePage={ jest.fn() } />)
    instance = wrapper.instance()
  });

  it('should match the snapshot', () => {

    expect(wrapper).toMatchSnapshot()
  });
});
