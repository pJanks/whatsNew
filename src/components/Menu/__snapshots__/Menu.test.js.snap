// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should match the snapshot 1`] = `
<aside>
  <button
    name="local"
    onClick={[Function]}
  >
    local
  </button>
  <button
    name="entertainment"
    onClick={[Function]}
  >
    entertainment
  </button>
  <button
    name="health"
    onClick={[Function]}
  >
    health
  </button>
  <button
    name="science"
    onClick={[Function]}
  >
    science
  </button>
  <button
    name="technology"
    onClick={[Function]}
  >
    technology
  </button>
</aside>
`;
